src/components/Pagination/Pagination.js

```diff
@@ -128,21 +127,33 @@ export class RawPagination extends Component {
   }
 
   render() {
-    const pager = new Paginator({
-      currentPage: this.props.currentPage,
-      total: this.props.nbPages,
-      padding: this.props.padding,
-    });
-
-    const createURL = this.props.createURL;
+    const {
+      createURL,
+      pages,
+      isFirstPage,
+      isLastPage,
+      currentPage,
+    } = this.props;
 
     return (
       <ul className={this.props.cssClasses.root}>
-        {this.props.showFirstLast ? this.firstPageLink(pager, createURL) : null}
-        {this.previousPageLink(pager, createURL)}
-        {this.pages(pager, createURL)}
-        {this.nextPageLink(pager, createURL)}
-        {this.props.showFirstLast ? this.lastPageLink(pager, createURL) : null}
+        {this.props.showFirstLast
+          ? this.firstPageLink({ isFirstPage, createURL })
+          : null}
```

Comment:
We could use short-circuit evaluation instead of having the ternary with the `null`.

Reply:
well yes 👍